diff -up kdecoration-master/CMakeLists.txt.1~ kdecoration-master/CMakeLists.txt
--- kdecoration-master/CMakeLists.txt.1~	2023-06-06 12:58:14.712366572 +0200
+++ kdecoration-master/CMakeLists.txt	2023-06-06 12:58:59.367649829 +0200
@@ -30,7 +30,7 @@ include(KDEGitCommitHooks)
 ecm_setup_version(${PROJECT_VERSION} VARIABLE_PREFIX KDECORATION2
                         VERSION_HEADER "${CMAKE_CURRENT_BINARY_DIR}/kdecoration2_version.h"
                         PACKAGE_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/KDecoration2ConfigVersion.cmake"
-                        SOVERSION 5)
+                        SOVERSION 6)
 
 #dependencies
 find_package(Qt6 ${QT_MIN_VERSION} CONFIG REQUIRED COMPONENTS
diff -up kdecoration-master/src/private/CMakeLists.txt.1~ kdecoration-master/src/private/CMakeLists.txt
--- kdecoration-master/src/private/CMakeLists.txt.1~	2023-06-02 13:34:04.000000000 +0200
+++ kdecoration-master/src/private/CMakeLists.txt	2023-06-06 12:58:14.712366572 +0200
@@ -28,7 +28,7 @@ target_link_libraries(kdecorations2priva
 target_include_directories(kdecorations2private INTERFACE "$<INSTALL_INTERFACE:${KDECORATION2_INCLUDEDIR}>" )
 
 set_target_properties(kdecorations2private PROPERTIES VERSION   ${KDECORATION2_VERSION}
-                                                      SOVERSION 10
+                                                      SOVERSION 11
                                                       EXPORT_NAME KDecoration2Private
 )
 
